src/frontends/tensorflow_common/src/op/matrix_inverse.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/inverse.hpp"
+
+#include "common_op_table.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+OutputVector translate_matrix_inverse_op(const NodeContext& node) {
+    /*
+    For matrix inverse translation, the goal is to convert the TensorFlow input tensor
+    into an OpenVINO OutputVector that the Inverse operation can accept.
+    */
+    // a function that performs standard or default checks on a specific operation within the OpenVINO pipeline
+    default_op_checks(node, 1, {"MatrixInverse"}); // copied from issue #22964
```

Comment:
```suggestion
    default_op_checks(node, 1, {"MatrixInverse"});
```

Reply:
we should not have GFI artefact in code

Reply:
I see, we must understand the function `default_op_checks`, which is to check the type of operation being performed. And be responsible for using the function here, so do not comment with `// copied from ...`.